ES/ESmr039.xml

```diff
@@ -145,18 +145,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <additions>
                      <list>
                         <item xml:id="a1">
-                           <locus target="#33vb">f. 33vb</locus>
-                           <desc>: Ownership note extended by an exhortation to the
-                              community.</desc>
-                           <quote xml:lang="gez">
-                              <locus target="#33vb">33vb</locus>: ዝመጽሐፈ፡ ጥምቀት፡ ዘመስጥሕ፡ ዘአባ፡ ሊባኖስ፡
-                              ዘአጽሐፍዋ፡ አቡነ፡ ወልደ፡ ተንሣኤ፡ ወቄስ፡ ገበዝ፡ አቡነ፡ ቄስ፡ ኢዮአብ፡ ምነው፡ አባቶቼ፡ ወንድሞቼ፡
-                              በምን፡ ነገር፡ ትጠሉኝ፡ አላቹህ፡ አሁን፡ አንድ፡ ቁምነገር፡ ሳልሰራ፡ አልቀርም፡ አትጥሉኝ።</quote>
-                           <quote xml:lang="en"/> Written partly in Gǝʿǝz and Amharic, possibly by
+                           <locus target="#33vb"/>
+                           <desc type="OwnershipNote">An ownership note extended with an exhortation to the
+                              community</desc>
+                           <q xml:lang="gez">
+                              ዝመጽሐፈ፡ ጥምቀት፡ ዘመስጥሕ፡ ዘአባ፡ ሊባኖስ፡
```

Comment:
something to mark up for Amharic?

Reply:
Do you mean under <langUsage>?

Reply:
I meant the description says Geez and Amharic and the quote says gez, so I was wondering, nevermind